aquadoggo/src/db/utils.rs

```diff
@@ -21,71 +24,64 @@ use crate::db::models::OperationFieldsJoinedRow;
 /// when retrieving an operation from the db.
 pub fn parse_operation_rows(
```

Comment:
Even though these parsing methods expect valid values to be coming from the dB, it would be nicer to catch and return errors, then unwrap the result when calling this method.

Reply:
Raised an issue: https://github.com/p2panda/aquadoggo/issues/244